textfield/internal/text-field.ts

```diff
@@ -604,6 +609,7 @@ export abstract class TextField extends textFieldBaseClass {
           aria-invalid=${this.hasError}
           aria-label=${ariaLabel}
           autocomplete=${autocomplete || nothing}
+          name=${this.name}
```

Comment:
Can you add `|| nothing` to not render the name attribute when none is provided?

```
name=${this.name || nothing}
```

Reply:
Thanks for the comments. PR updated as advised.